recipes/festlex-cmu/meta.yaml

```
@@ -0,0 +1,34 @@
+{% set name = "festlex-cmu" %}
+{% set version = "2.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://festvox.org/packed/festival/2.5/festlex_CMU.tar.gz
+  sha256: c19430919bca45d5368cd4c82af6153fbcc96a487ebd30b78b5f3c08718b7c07
+
+build:
+  number: 0
+  noarch: generic
+
+test:
+  commands:
+    - test -f $PREFIX/share/festival/dicts/cmu/cmulex.scm  # [linux]
+    - test -f $PREFIX/share/festival/dicts/cmu/cmudict-0.4.out  # [linux]
+    - test -f $PREFIX/share/festival/dicts/cmu/cmu_lts_rules.scm  # [linux]
```

```suggestion
build:
  number: 0
  noarch: generic

requirements:
  run:
    - __unix  # [unix]

test:
  commands:
    - test -f $PREFIX/share/festival/dicts/cmu/cmulex.scm  # [unix]
    - test -f $PREFIX/share/festival/dicts/cmu/cmudict-0.4.out  # [unix]
    - test -f $PREFIX/share/festival/dicts/cmu/cmu_lts_rules.scm  # [unix]
```
You probably want to structure your noarch packages like this because the UNIX prefix and Windows prefix are different. This way, you will be able to add support for MacOS without rebuilding these data packages.



https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-packages-with-os-specific-dependencies

---

@carterbox How did you tell the unix and windows prefixes are different? The path to the data files is configured in the conda-forge build files. In the unlikely event that somebody adds windows support to festival feedstock, he could easily make sure these paths are the same on Windows.

---

Windows packages are installed to `%LIBRARY_PREFIX%` i.e. `%PREFIX%\Library`. I don't know why. It just is that way.

https://docs.conda.io/projects/conda-build/en/stable/user-guide/environment-variables.html#environment-variables-set-during-the-build-process
https://docs.conda.io/projects/conda-build/en/stable/resources/use-shared-libraries.html#shared-libraries-in-windows